datasets/createljspeech.py

```diff
@@ -0,0 +1,47 @@
+# This script generates a folder structure for ljspeech-1.1 processing from mimic-recording-studio database
+# Written by Thorsten Mueller (MrThorstenM@gmx.net) on november 2019 without any warranty
+
+import argparse
+import sqlite3
+import os
+from shutil import copyfile
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--out_dir', default=os.path.expanduser('~/tacotron'))
+    parser.add_argument('--mrs_dir', required=True)
+    args = parser.parse_args()
+
+    dir_base_ljspeech = os.path.join(args.out_dir,"LJSpeech-1.1")
+    dir_base_ljspeech_wav = os.path.join(dir_base_ljspeech,"wavs")
+    dir_base_mrs = args.mrs_dir
+    os.makedirs(dir_base_ljspeech_wav, exist_ok=True)
+
+    conn = sqlite3.connect(os.path.join(dir_base_mrs,"backend","db","mimicstudio.db"))
+    c = conn.cursor()
+
+    # Get user id from sqlite to find recordings in directory structure
+    # TODO: Currently just works with one user
+    for row in c.execute('SELECT uuid FROM usermodel LIMIT 1;'):
```

Comment:
I thought about adding an optional parameter --username. If it's passed the correct guid is selected from the sqlite db. If it's not passed the first user from database is chosen (as it's currently). Then it would be possible to export the correct dataset, even if one mimic-recording-studio database is used by more than one reader.

@krisgesling What do you think on that.